emitter/redshift/emitter.go

```diff
@@ -50,6 +54,12 @@ func (e RedshiftEmitter) Emit(s3Key string, b io.ReadSeeker) {
 			break
 		}
 	}
+
+	if err != nil {
```

Comment:
Yeah. In go, it is specified to return an explicit nil.

Ref: https://golang.org/doc/faq#nil_error